<<<<<<< HEAD
#Service to set the communication mode of the robot

int64 mode  #1-Blocking; 0-Nonblocking
---
int64 ret
=======
#Service to set the communication mode of the robot

int64 mode  #1-Blocking; 0-Nonblocking
---
int64 ret
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
string msg